flink-runtime/src/test/resources/job-details/job-details-compatible.js

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Maybe just move this into a static string so we don't have to bother with a license header.